exporters/prometheus/exporter_test.go

```diff
@@ -431,6 +432,47 @@ func TestPrometheusExporter(t *testing.T) {
 				counter.Add(ctx, 5, otelmetric.WithAttributeSet(attrs2))
 			},
 		},
+		{
+			name:         "monotonic sum with exemplars",
```

Comment:
```suggestion
			name:         "non-monotonic sum does not add exemplars",
```

This is actually testing an updown (non-monotonic) counter, and we are checking that it does not add exemplars.  Consider renaming the test file as well.